addons/account/models/chart_template.py

```diff
@@ -243,7 +243,7 @@ def _pre_reload_data(self, company, template_data, data):
                     journal = self.env['account.journal'].with_context(active_test=False).search([
```

Comment:
`search_count` would be more efficient than `search` in this situation, where a limit is actually set (@Levizar explains it with talent, just sayin, just in case)

Reply:
But we use the journal :confused: